src/routes/search/containers/SearchPage.js

```diff
@@ -13,8 +13,9 @@ import type { StateType } from '../../../modules/app/StateType'
 import SearchFeedback from '../components/SearchFeedback'
 import type { LocationState } from 'redux-first-router'
 import normalizeSearchString from '../../../modules/common/utils/normalizeSearchString'
+import { fromString as htmlToText } from 'html-to-text'
 
-type CategoryListItemType = {| model: CategoryModel, subCategories: Array<CategoryModel> |}
+type CategoryListItemType = {| model: CategoryModel, contentWithoutHtml?: string, subCategories: Array<CategoryModel> |}
```

Comment:
Not sure about this:
```suggestion
type CategoryentryType = {| model: CategoryModel, contentWithoutHtml?: string, subCategories: Array<CategoryModel> |}
```